docs/CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Remove "resolution" field from Flaw (OSIDB-2737)
 - Remove several cvss fields from Flaw (OSIDB-2749)
 - Remove several cvss fields from Affect (OSIDB-2749)
+- Remove "statypete" field from Flaw Comment (OSIDB-2745)
```

Comment:
typo `statypete`

Reply:
also use the model name, i.e. `FlawComment`

Reply:
Good catch! changed!